### PR TITLE
ECO-792: Implement parser for simple types of StoredValue

### PR DESCRIPTION
**Pull request by AbnerZheng**
### Overview
This PR implement deserializer for kinds of StoredValue except for CLValue, including simple unit tests.

### Which JIRA ticket does this PR relate to?
https://casperlabs.atlassian.net/browse/ECO-792

### Complete this checklist before you submit this PR

- [X] This PR contains no more than 200 lines of code, excluding test code.
- [X] This PR meets [CasperLabs coding standards](https://casperlabs.atlassian.net/wiki/spaces/EN/pages/16842753/Coding+Standards).
- [X] If this PR adds a new feature, it includes tests related to this feature.
- [X] You assigned one person to review this PR.
- [X] Your GitHub account is linked with our [Drone CI](https://drone-auto.casperlabs.io/) system. This is necessary to run tests on this PR.
- [X] Do not forget to run `bors r+` if GitHub policy is not enforced, e.g. when merging into another feature branch. It may be omitted under some circumstances if this PR intentionally assumes that integration tests will fail but will be fixed with the future PRs.

### Notes

_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._

### CODE REVIEW

File: packages/sdk/test/lib/StoredValue.test.ts [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        action_thresholds: {
+          deployment: 1,
+          key_management: 1
+        }
+      }
+    };
+
+    const serializer = new TypedJSON(StoredValue);
+    const storedValue = serializer.parse(mockJson);
+    console.log(storedValue);
```
Comment (zie1ony) [inline packages/sdk/test/lib/StoredValue.test.ts]: Remove this log.

File: packages/sdk/test/lib/StoredValue.test.ts [hunk 2]
```diff
@@ -0,0 +1,132 @@
+import { TypedJSON } from 'typedjson';
+import { StoredValue } from '../../src/lib/StoredValue';
+import { expect } from 'chai';
+
+describe('StoredValue', () => {
+  it('should parse Account stored value correctly', function () {
```
Comment (zie1ony) [inline packages/sdk/test/lib/StoredValue.test.ts:6]: This test doesn't test anything. There should be some asserts at the end of the test, right?

File: packages/sdk/test/lib/StoredValue.test.ts [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    const mockJson = {
+      Contract:
+        'ea058d32053f59e9f66dd3d4de4594a8a3de36c65c87417efe79cdc7c1b926b4b1575b26a474dc52c03c2a7207db951fa4f82214ee4257c4b99bbab603908d5900000000010000000b00000063616c6c5f6661756365740b00000063616c6c5f66617563657402000000060000007461726765740f2000000006000000616d6f756e7408090100010000000000000000000000'
+    };
+
+    const serializer = new TypedJSON(StoredValue);
+    const storedValue = serializer.parse(mockJson);
+    expect(storedValue?.Contract).to.not.eq(undefined);
+    expect(storedValue?.Contract).to.eq(mockJson.Contract);
+    console.log(storedValue);
```
Comment (zie1ony) [inline packages/sdk/test/lib/StoredValue.test.ts]: Remove the log.

File: packages/sdk/test/lib/StoredValue.test.ts [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+    const mockJson = {
+      ContractPackage:
+        '9eab12b986299509b4471060fe4d17f087bdd2596871c38d39019ef94f8d10a6070100000001000000010000001c16234ad1d27b51614ec5dca0bc28ea235eb2dc3a1f9d98aa238dc3df1fd63a0000000000000000'
+    };
+
+    const serializer = new TypedJSON(StoredValue);
+    const storedValue = serializer.parse(mockJson);
+    expect(storedValue?.ContractPackage).to.not.eq(undefined);
+    expect(storedValue?.ContractPackage).to.eq(mockJson.ContractPackage);
+    console.log(storedValue);
```
Comment (zie1ony) [inline packages/sdk/test/lib/StoredValue.test.ts]: Remove the log.

File: packages/sdk/test/lib/StoredValue.test.ts [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+    };
+
+    const serializer = new TypedJSON(StoredValue);
+    const storedValue = serializer.parse(mockJson);
+    expect(storedValue?.DeployInfo).to.not.eq(undefined);
+    expect(storedValue?.DeployInfo?.deployHash).to.eq(
+      mockJson.DeployInfo.deploy_hash
+    );
+    expect(storedValue?.DeployInfo?.from).to.eq(mockJson.DeployInfo.from);
+    console.log(storedValue);
```
Comment (zie1ony) [inline packages/sdk/test/lib/StoredValue.test.ts]: Remove the log.

**Review by zie1ony — APPROVED** [review]
Looks good. Just minor changes needed.

File: packages/sdk/test/lib/StoredValue.test.ts (hunk 2 above)
Comment (AbnerZheng) [reply packages/sdk/test/lib/StoredValue.test.ts:6]: Yes, you're right. Done.

File: packages/sdk/test/lib/StoredValue.test.ts (hunk 1 above)
Comment (AbnerZheng) [reply packages/sdk/test/lib/StoredValue.test.ts]: Done.

File: packages/sdk/test/lib/StoredValue.test.ts (hunk 4 above)
Comment (AbnerZheng) [reply packages/sdk/test/lib/StoredValue.test.ts]: Done.

File: packages/sdk/test/lib/StoredValue.test.ts (hunk 5 above)
Comment (AbnerZheng) [reply packages/sdk/test/lib/StoredValue.test.ts]: Done.

**Pull request merged**